$OpenBSD: patch-src_player_c,v 1.21 2016/11/05 21:27:26 dcoppa Exp $

Turn off ffmpeg's non-fatal error messages.

"Skipping 0 bytes of junk" is useless to the user, and essentially
indicates a NOP.

upstream git commit 7c2d9ebb7f61cf04d75e28ca5cd6932002c4357b

--- src/player.c.orig	Sat Nov  5 22:21:55 2016
+++ src/player.c	Sat Nov  5 22:23:03 2016
@@ -68,6 +68,7 @@ static void printError (const BarSettings_t * const se
  */
 void BarPlayerInit () {
 	ao_initialize ();
+	av_log_set_level (AV_LOG_FATAL);
 	av_register_all ();
 	avfilter_register_all ();
 	avformat_network_init ();
